### ISSUE TITLE
multiple haze effects

### ISSUE THREAD

**Issue by Kyant0** [chrisbanes/haze#30]
I want to blur a screen with multiple Rects with different haze background colors.
I found it impossible to put multiple haze modifiers in one composable, but it works with wrapper layouts with only one haze modifier of each.

Is there convenient API that provides multiple complex haze effects in one modifier? For example:
```kotlin
haze(
  Haze(rect1, bg1, tint1),
  Haze(rect2, bg2, tint2),
)
```

Additionally, both `Rect` and `RoundRect` can be combined.

**Comment by chrisbanes** [comment]
Mostly released in 0.4.0

**Comment by chrisbanes** [comment]
Final bit of this was added in #81 in 0.4.3
